packages/cadl-ranch-specs/http/spread/main.tsp

```diff
@@ -0,0 +1,158 @@
+import "@typespec/http";
+import "@azure-tools/cadl-ranch-expect";
+
+using TypeSpec.Http;
+
+@doc("Test for the spread operator.")
+@supportedBy("dpg")
+@scenarioService("/spread")
+namespace Spread;
+
+@doc("This is a simple model.")
+model ModelBodyParameter {
+  name: string;
+}
+
+@scenario
+@scenarioDoc("""
+Test case for spread named model. Should generate request body model named ModelBodyParameter.
```

Comment:
Just a confirmation, we expect `ModelBodyParameter` model, as when it spread, it does not spread a alias, right?

Reply:
yeah for `model`s we should generate them, even if they're spread. we should never generate an alias as a model